src/pages/integration-builder/builder/unitySnippets/get-login-code.ts

```diff
@@ -0,0 +1,13 @@
+export const getLoginCode = (isMFA: boolean, isCurve?: boolean) => {
+  let mfaCode = "";
+  let curveCode = "";
+  if (isMFA) {
+    mfaCode = `
+            mfaLevel = MFALevel.MANDATORY`;
+  }
+  if (isCurve) {
```

Comment:
Are we using this in the integration builder? I don't see any toggle.

Reply:
This will be added after the highlighting issue is fixed. Approving this for now.